accelerometer/summariseEpoch.py

```diff
@@ -356,12 +359,17 @@ def calculateECDF(e, inputCol, summary, useRecommendedImputation):
     :return: Write dict <summary> keys '<inputCol>-ecdf-<level...>mg'
     :rtype: void
     """
-
-    ecdf1, step = np.linspace(1, 20, 20, retstep=True)  # 1mg bins from 1-20mg
-    ecdf2, step = np.linspace(25, 100, 16, retstep=True)  # 5mg bins from 25-100mg
-    ecdf3, step = np.linspace(125, 500, 16, retstep=True)  # 25mg bins from 125-500mg
-    ecdf4, step = np.linspace(600, 2000, 15, retstep=True)  # 100mg bins from 500-2000mg
-    ecdfXVals = np.concatenate([ecdf1, ecdf2, ecdf3, ecdf4])
+    if enhanced:
+        ecdf1, step = np.linspace(1, 100, 100, retstep=True)  # 1mg bins from 1-100mg
+        ecdf2, step = np.linspace(105, 500, 80, retstep=True)  # 5mg bins from 105-500mg
+        ecdf3, step = np.linspace(525, 2000, 60, retstep=True)  # 25mg bins from 125-500mg
+        ecdfXVals = np.concatenate([ecdf1, ecdf2, ecdf3])
+    else: 
+        ecdf1, step = np.linspace(1, 20, 20, retstep=True)  # 1mg bins from 1-20mg
+        ecdf2, step = np.linspace(25, 100, 16, retstep=True)  # 5mg bins from 25-100mg
+        ecdf3, step = np.linspace(125, 500, 16, retstep=True)  # 25mg bins from 125-500mg
+        ecdf4, step = np.linspace(600, 2000, 15, retstep=True)  # 100mg bins from 500-2000mg
+        ecdfXVals = np.concatenate([ecdf1, ecdf2, ecdf3, ecdf4])
```

Comment:
I dont know about this code but what do you think about keeping just the "enhanced" version? It seems strange to have both `--intensityDistribution`  and `--intensityDistributionEnhanced`

Reply:
Have now changed the implementation to generalise it, thanks for pushing me to do that!